models/Product.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+var URLSlug = require("mongoose-slug-generator");
+const mongoosePaginate = require("mongoose-paginate");
+mongoose.plugin(URLSlug);
+const ProductSchema = new mongoose.Schema(
+  {
+    title: { type: String, required: true, unique: true },
+    desc: { type: String, required: true },
+    images: [Object],
+    categories: { type: mongoose.Schema.Types.ObjectId, ref: "Category" },
```

Comment:
since it is singular..name should also needs to change

Reply:
done